sdk/liftrqumulo/Azure.ResourceManager.LiftrQumulo/api/Azure.ResourceManager.LiftrQumulo.netstandard2.0.cs

```diff
@@ -0,0 +1,480 @@
+namespace Azure.ResourceManager.LiftrQumulo
```

Comment:
project, folders, etc all need to update to be the approved name `Azure.ResourceManager.Qumulo`

Reply:
changed to `Azure.ResourceManager.Qumulo`